flink-job/src/main/java/com/ververica/field/dynamicrules/sinks/AlertsSink.java

```diff
@@ -50,7 +51,7 @@
         String alertsTopic = config.get(ALERTS_TOPIC);
         return new FlinkKafkaProducer011<>(alertsTopic, new SimpleStringSchema(), kafkaProps);
       case PUBSUB:
-        return PubSubSink.<String>newBuilder()
+        return PubSubSink.newBuilder()
```

Comment:
Revert. 
I am pretty sure that this does not actually work without the type when you try to use google pubsub connector.

Reply:
I doubted IntelliJ for a second, but the class' generic type argument only applies to non-static members, but `org.apache.flink.streaming.connectors.gcp.pubsub.PubSubSink#newBuilder` does not have and type parameters on its own.

I don't have a google pubsub to test this against, but I suppose, **if** this needs to be added, then not for the `newBuilder` method...
(on the other hand, you kind of make the type clear by using `SimpleStringSchema`)

Reply:
I do not have time  to get into the details of what is happening under the hood at the moment, but remember this failing. Unless you can test with PubSub and make sure that it works, I would propose to skip this change.

Reply:
ok, let's keep it then